processor/intervalprocessor/factory.go

```diff
@@ -23,7 +24,10 @@ func NewFactory() processor.Factory {
 }
 
 func createDefaultConfig() component.Config {
-	return &Config{}
+	return &Config{
+		Interval:     15 * time.Second,
+		MaxStaleness: 0,
```

Comment:
How does this compare to work on delta-to-cumulative? I think we should closely match the design there, since many of the cache management concerns should be the same. If necessary, it may make sense to wait until the relevant fields are locked in on the other processor, then extract them into a shared package.

Reply:
DeltaToCumulative hasn't yet implemented MaxStaleness: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/30707#discussion_r1469598817

I based the current defaults on the existing CumulativeToDelta values